Makefile

```diff
@@ -2,7 +2,7 @@
 CC        = clang -std=c99
 CPPFLAGS  = -MMD -MP -DSYNTH_LIB_ALONE
 CFLAGS    = -Wall -Wextra -pedantic -O3
-LDFLAGS   = 
+LDFLAGS   = -lm
```

Comment:
This is very helpful.
I really didn't understand before why there has to be the `-lm` flag. Thanks!
[Why do you have to link the math library in C?](https://stackoverflow.com/questions/1033898/why-do-you-have-to-link-the-math-library-in-c)